test/dotnet/input/Datatypes/TestTvp.txt

```diff
@@ -30,4 +31,63 @@ create schema this_schema_name_is_also_very_looooooooooooooooooooooooooooooooooo
 create type this_schema_name_is_also_very_loooooooooooooooooooooooooooooooooooooooooong.this_tvp_type_name_is_very_loooooooooooooooooooooooooooooooooooooooooong as table (a int, b smallint, c bigint, d tinyint, e bit, f char(10), g nchar(10), h varchar(10), i nvarchar(10), l varbinary(10), m binary(10), n date, o datetime, p money, q uniqueidentifier,r float, s real, t numeric(4,3), u decimal(5,3), v time(5), w datetime2(5))
 prepst#!#Select * from @this_tvp_parameter_name_is_very_loooooooooooooooooooooooooooooooooooooooooong#!#tvp|-|this_tvp_parameter_name_is_very_loooooooooooooooooooooooooooooooooooooooooong|-|this_schema_name_is_also_very_loooooooooooooooooooooooooooooooooooooooooong.this_tvp_type_name_is_very_loooooooooooooooooooooooooooooooooooooooooong|-|../../../utils/tvp-dotnet.csv
 drop type this_schema_name_is_also_very_loooooooooooooooooooooooooooooooooooooooooong.this_tvp_type_name_is_very_loooooooooooooooooooooooooooooooooooooooooong
-drop schema this_schema_name_is_also_very_loooooooooooooooooooooooooooooooooooooooooong
\ No newline at end of file
+drop schema this_schema_name_is_also_very_loooooooooooooooooooooooooooooooooooooooooong
+
+create table t(a int)
+create type t1 as table (a int)
+create procedure proc1 @t2 as t1 readonly as begin set nocount on; insert into t (a) select a from @t2 end
+storedproc#!#prep#!#proc1#!#tvp|-|t2|-| |-|../../../utils/tvp-test-dotnet.csv
```

Comment:
This is the same tests which were present and does not test the current use case in mind, How are we testing the change you have made?

Reply:
I am not passing the typeName "t1" explicitly unlike the tests above but rather an one space char which is trimmed in dotnet.